lib/desired-caps.js

```diff
@@ -1,10 +1,24 @@
-import _ from 'lodash';
 import { commonCapConstraints } from 'appium-android-driver';
 
-let desiredCapConstraints = _.clone(commonCapConstraints);
-desiredCapConstraints.app = {
-  presence: true,
-  isString: true,
+let selendroidCapConstraints = {
+  app: {
+    presence: true,
+    isString: true,
+  },
+  automationName: {
```

Comment:
why is automationName required for selendroid? if you're just running this driver on its own, is it required?